packages/presentation/lib/src/pages/movie_details_page/bloc/movie_details_bloc.dart

```diff
@@ -71,7 +72,13 @@ class _MovieDetailsBloc extends BlocImpl<BaseArguments, DetailsData>
   }
 
   @override
-  void share(String message) => ShareText.shareText(message);
+  void share(String message, {RenderBox? renderBox}) {
+    if (renderBox != null) {
+      ShareText.macShareText(message, renderBox);
+    } else {
+      ShareText.shareText(message);
+    }
```

Comment:
why do u need to pass the renderbox?

Reply:
it's for macOs plugin, to know where to draw dropDown menu

Reply:
strange. check plz how is it implemented in https://pub.dev/packages/share_plus. seems that they don't have the param. but if there's no other way - ok

Reply:
///The optional [sharePositionOrigin] parameter can be used to specify a global
  /// origin rect for the share sheet to popover from on iPads and Macs. It has no effect
  /// on other devices. (c) GitHub

share_plus requires iPad users to provide the sharePositionOrigin parameter.
Without it, share_plus will not work on iPads and may cause a crash or letting the UI not responding.
To avoid that problem, provide the sharePositionOrigin. (c) pub.dev

I corrected plugin to be able work without render box, but in project I decided to ask render box to avoid problems